Jenkinsfile.github

```
@@ -0,0 +1,42 @@
+@Library('csm-shared-library') _
+
+pipeline {
+    agent {
+        label "metal-gcp-builder"
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: "10"))
+        timestamps()
+    }
+
+    environment {
+        NAME = "cray-vpa"
+        DESCRIPTION = "Deploys the Oauth2 Proxy ingress to the Cray system."
```

Not building the Oauth2 proxy chart
```suggestion
        DESCRIPTION = "Deploys the vpa policy to CSM systems."
```